subscribers:
    bounding_boxes:
        topic: /darknet_ros/bounding_boxes
        queue_size: 1

    depth_map:
        topic: /camera/depth/image_rect_raw
        queue_size: 10

publishers:
    object_data:
        topic: /yolo_depth_fusion/objects
        queue_size: 1
        latch: false

settings:
    depth_filter: 1.5
    threshold: 0.3
